intranet/static/js/eighth/schedule.js

```diff
@@ -163,4 +161,4 @@ $(function() {
             });
         }
     });
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2746691966/issues/source?bid=3176160&fileBranchId=3355598#l164)